legal-api/src/legal_api/models/filing.py

```diff
@@ -58,10 +58,10 @@ class Status(Enum):
 
     id = db.Column(db.Integer, primary_key=True)
     _completion_date = db.Column('completion_date', db.DateTime(timezone=True))
-    _filing_date = db.Column('filing_date', db.DateTime(timezone=True), default=datetime.utcnow)
+    _filing_date = db.Column('filing_date', db.DateTime(timezone=True), default=datetime.now)
```

Comment:
Are there unit tests for these?

Reply:
Lekshmi answered that, yes, there is a unit test that check for the existence of of this property, but it does not validate the time (which is kind of hard when you use "now", since it's always changing). So I'm OK with the status quo.